src/main/java/com/smalaca/messagesender/service/MessageCrud.java

```diff
@@ -27,4 +28,14 @@ public Response createNew(MessageDto messageDto) {
 
         return Response.aFailureResponse("Message already exists");
     }
+
+    public Response deleteMessage(String messageId) {
+        if (messageId.equals("NO_ID")) return Response.aFailureResponse("Requested parameter 'id' to proceed.");
```

Comment:
i tutaj wasz serwis dowiaduje się o istnieniu kontrollera. Jeżeli usuniecie default value, to ten kawałek również można wyrzucić